main.go

```diff
@@ -115,15 +128,10 @@ func main() {
 }
 
 func segment(s *bytes.Buffer) string {
-	b := bytes.NewBufferString("\"\" +\n")
-	for s.Len() > 0 {
-		v := string(s.Next(100))
-		b.WriteString(fmt.Sprintf("\t\t\t%q", v))
-		if s.Len() > 0 {
-			b.WriteString(" +\n")
-		}
-	}
-	return b.String()
+	var b bytes.Buffer
+	b64 := base64.NewEncoder(base64.URLEncoding, &b)
+	b64.Write(s.Bytes())
```

Comment:
Rather, encode the entire buffer into base64, then break it up into 100-length lines. This will make all the lines the same length, which I would like.